### ISSUE TITLE
MIT compatible license?

### ISSUE THREAD

**Issue by kevinbader** [launchbadge/realworld-axum-sqlx#8]
This repository is a great source for best practices. It'd be cool to re-use some parts of it as-is, like the SQL function for adding the `set_updated_at` trigger. However, the license is quite restrictive; I'm definitely not a license expert, but it appears to me I can't copy over that trigger into my MIT licensed FOSS project.

I realize this might be intended, but on the other hand being a source to copy from might also just be the reason this repo exists. So I'm curious :)

**Comment by JonasKs** [comment]
I agree, this license seems very restrictive for the nature of this project, @abonander? 😊

**Comment by JonasKs** [comment]
I see the license was set because of [this](https://www.reddit.com/r/rust/comments/shetb1/show_rrust_a_rust_implementation_of_the_realworld/hv3b330/):

> The main concern was someone taking it and using it as the base for a proprietary project, so a copyleft license made sense to us. 

Weird, considering their `sqlx` is MIT licensed, but not this, which is literally intended as inspiration/for teaching purposes 😁
